extensions/geode-modules/src/main/java/org/apache/geode/modules/util/RegionConfigurationCacheListener.java

```diff
@@ -99,14 +97,12 @@ public void afterRegionCreate(RegionEvent<String, RegionConfiguration> event) {
       }
 
     }
+
     if (this.cache.getLogger().fineEnabled()) {
       this.cache.getLogger().fine(builder1.toString());
-      if (builder2 != null) {
+      if (builder2.length() != 0) {
         this.cache.getLogger().fine(builder2.toString());
       }
     }
   }
-
-  @Override
-  public void init(Properties p) {}
```

Comment:
I assume we can get rid of this because the super also does nothing?

Reply:
Indeed, the `Declarable` interface has the method declared as `default void init(Properties props) {};`.